### PR TITLE
little change

### PR DESCRIPTION
**Pull request by sold877**
please check it

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -25,11 +25,11 @@
 		echo "Name of student =".$name.". and his id is =".$id;
 
 
-		$name="RMishu";
+		$name="Mishu";
```
Comment (faysalmazeddiu) [inline index.php:28]: can you change also id

File: index.php [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 
 		$name="Mishu";
-		$id="112-35-204";
+		$id="112-35-20489";
```
Comment (faysalmazeddiu) [inline index.php:29]: can you add another id

File: index.php [hunk 3]
```diff
@@ -27,6 +27,7 @@
 
 		$name="Mishu";
 		$id="112-35-20489";
+		$id="112-35-20000";
```
Comment (faysalmazeddiu) [inline index.php:30]: thanks

Comment (sold877) [reply index.php:30]: thanks too

**Pull request merged**